backend/src/database.ts

```diff
@@ -0,0 +1,22 @@
+import { Sequelize } from "sequelize";
+
+const DEFAULT_PG_DATABASE = "eight_bar_line_db";
+const DEFAULT_PG_USERNAME = "user";
+const DEFAULT_PG_PASSWORD = "mysecretpassword";
```

Comment:
To clarify, this info needs to match the info I provided to `./backend/database.env`